c_emulator/riscv_sim.c

```diff
@@ -414,7 +452,6 @@ static int process_args(int argc, char **argv)
     case OPT_TRACE_OUTPUT:
       trace_log_path = optarg;
       fprintf(stderr, "using %s for trace output.\n", trace_log_path);
-      break;
```

Comment:
why is this line gone?

Reply:
Good spot - probably a rebase mistake. I'll fix it.

Reply:
Fixed.